packages/atomic/src/components/result-template-components/atomic-result-text/atomic-result-text.tsx

```diff
@@ -29,6 +29,11 @@ export class AtomicResultText {
    */
   @Prop() public shouldHighlight = true;
 
+  /**
+   * The non-localized text to display if the field has no value.
+   */
+  @Prop() public default?: string;
```

Comment:
Could we set the default to `noTitle`? I don't think devs will know about it otherwise, and would only think of configuring this value when they see the issue in the UI.

Reply:
Same as I answered here https://github.com/coveo/ui-kit/pull/636#discussion_r596014998
This is the atomic-result-text component, which renders the text value of a field. Most often, when the value is not available, we want that component removed. In other cases, there is the default. There is no link with this component with the title property or any specific property.

For the atomic-result-link, it uses the atomic-result-text with the title key as default. In the other PR, the result link is slottable, and the default will be an atomic-result-text with title.